implementations/swift/Makefile

```diff
@@ -10,11 +14,11 @@ xcode_build:
 xcode_signed_build:
 	xcodebuild archive -project ockam/ockam_app/Ockam.xcodeproj/ -scheme "Portals, by Ockam" -configuration Release -archivePath build/Ockam.xcarchive -arch $(ARCH)  CODE_SIGN_IDENTITY="${CODE_SIGN_IDENTITY}"  PROVISIONING_PROFILE_SPECIFIER="${PROVISIONING_PROFILE_SPECIFIER}" DEVELOPMENT_TEAM="${DEVELOPMENT_TEAM}"
 ockam_app_lib_release:
-	$(MAKE) -f ../rust/Makefile build_release_ockam_app_lib
+	$(MAKE) -f ../rust/Makefile build_release
```

Comment:
wouldn't be easier to move the `ockam_command_release` dependency to `xcode_build`?

Reply:
I also think it's a bit convoluted. I refactored it (see [3c2be02](https://github.com/build-trust/ockam/pull/7200/commits/3c2be027a6ea9ad0ec06f1ec92e004f5f4f7eaa7)), tell me what you think.